docs/UserGuide.md

```diff
@@ -1,226 +1,23 @@
-
-_Details coming soon ..._
-
---------------------------------------------------------------------------------------------------------------------
-
-## FAQ
+### Examples:
+- ```add n/John Wang p/98765432 e/johnwang@example.com a/John street, block 123, #01-01 d/20.03.2000```
+- ```add n/John p/92781123 e/john@example.com a/Donald street, block 248, #02-04 d/20 MAR 2000```
+- ```add n/Charlotte p/81286623 e/charlotte@example.com a/Charity street, block 101, #10-82 d/March 20, 2000```
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+## Optional Parameter 2: Date of Birth
 
---------------------------------------------------------------------------------------------------------------------
+### Format: ```edit INDEX [n/NAME] [d/DOB]```
 
-## Command summary
+- Edits the date of birth of a person in the contact list. Date formats accepted are: d.M.yyyy, dd MMM yyyy, M d, yyyy
+- INDEX must be positive integers (i.e 1,2,3…)
+- INDEX must be within the range of the contact list index (i.e from 1 to size of contact list).
 
-| Action     | Format, Examples                                                                                                                                                      |
-|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague` |
-| **Clear**  | `clear`                                                                                                                                                               |
-| **Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                   |
-| **Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                           |
-| **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                            |
-| **List**   | `list`                                                                                                                                                                |
-| **Help**   | `help`                                                                                                                                                                |
+### Examples:
+- ```edit 1 d/20.03.2000```
+- ```edit 2 d/20 MAR 2000```
+- ```edit 3 d/March 20, 2000```
```

Comment:
A newline would be necessary here to prevent codecov from failing!

Reply:
i see thank you for letting me know